### ISSUE TITLE
Deploy: First time (staging) setup issues

### ISSUE THREAD

**Issue by dgrdl** [flyntwp/flynt-cli#27]
When setting up a project, the `.env` and `web/.htaccess` files (among others) are excluded from deployment by default, which leads to the need for manual setup on the server. That entails:

- creating a .env file / copying the existing from local and adjusting it
- logging into the backend via `your-website.domain/wp/wp-admin` and opening the permalinks page

I'm not sure if there is a nicer way, but maybe we could (optionally) deploy once after the create task has finished? This step could then take care of creating the .env file and flushing the rewrite rules.

The annoying thing would be the increase in prompts you have to go through (but you need to go through these anyways with our workflow).

**Comment by bdbch** [comment]
> i also do not want to have a command, that if you run it accidentally, will delete something you have set up on a remote server (could be production as well).

agree, did this once accidentally via flynt deploy with a wrong dir setup in flynt.json. 

After setting up multiple flynt websites the initial setup became less and less annoying with each project.
